### DISCUSSION TITLE
Upgrade from 2.4.30 to 2.4.40 not working still shows .30 and no new integrations

### DISCUSSION THREAD

**Discussion by Phillip-Samson** [Security-Onion-Solutions/securityonion#12193] [2.4] [answered]
### Version

2.4.30

### Installation Method

Security Onion ISO image

### Description

upgrading

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Exceeds minimum requirements

### CPU

16

### RAM

64G

### Storage for /

500G

### Storage for /nsm

3TB

### Network Traffic Collection

other (please provide detail below)

### Network Traffic Speeds

more than 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

Yes, there are additional clues in /opt/so/log/ (please provide detail below)

### Detail

`### Preparing soup at Wed Jan 17 01:37:31 PM UTC 2024 ###

Checking if we can access the salt master and that it is ready at: Jan 17 13:37:31
Successfully accessed and salt master ready at: Jan 17 13:37:37
Checking to see if this is a manager.

This is a manager, so we can proceed.

The manager's pillars can be rendered. We can proceed with SOUP.

Checking to see if this is an airgap install.

Found that Security Onion 2.4.30 is currently installed.

Cloning Security Onion github repo into /tmp/sogh/securityonion.
Removing previous upgrade sources.
Cloning the Security Onion Repo.
Cloning into 'securityonion'...
remote: Enumerating objects: 84580, done.
remote: Counting objects: 100% (1522/1522), done.
remote: Compressing objects: 100% (592/592), done.
remote: Total 84580 (delta 1003), reused 1410 (delta 912), pack-reused 83058
Receiving objects: 100% (84580/84580), 47.29 MiB | 9.15 MiB/s, done.
Resolving deltas: 100% (56286/56286), done.
Verifying we have the latest soup script.
This version of the soup script is up to date. Proceeding.
Let's see if we need to update Security Onion.
Checking to see if there are hotfixes needed
You are already running the latest version of Security Onion.
`

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by TOoSmOotH** [comment]
2.4.40 has not been released yet.

**Comment by TOoSmOotH** [comment, marked as answer]
https://github.com/Security-Onion-Solutions/securityonion/discussions/12198
